Tasks/NuGetInstallerV0/nugetinstaller.ts

```diff
@@ -76,7 +76,7 @@ async function main(): Promise<void> {
         // locateNuGetExe() will strip them and check for existence there.
         let nuGetPath = tl.getPathInput("nuGetPath", false, false);
         let userNuGetProvided = false;
-        if(nuGetPath !== null && tl.filePathSupplied("nuGetPath")){
+        if(nuGetPath !== undefined && tl.filePathSupplied("nuGetPath")){
```

Comment:
I'd rather change it to `nuGetPath != null` to check for both `null` and `undefined` just in case

Reply:
Hi @ByAgenT. Thank you for the suggestion. I have made the change as suggested and pushed it to the PR.